staging/src/k8s.io/client-go/tools/cache/index_test.go

```diff
@@ -88,7 +88,7 @@ func TestMultiIndexKeys(t *testing.T) {
 				found.Insert(item.(*v1.Pod).Name)
 			}
 			items := v.List()
-			if !found.HasAll(items...) {
+			if found.Len() != len(items) || !found.HasAll(items...) {
```

Comment:
could simplify to `found.Equal(expected)`

Reply:
https://github.com/kubernetes/kubernetes/blob/08d380e3b9964a52872042dd61d4c82390a5763f/staging/src/k8s.io/apimachinery/pkg/util/sets/string.go#L159-L161

Fixed.